lib/src/main/java/com/fabernovel/statefullayout/State.kt

```diff
@@ -57,17 +59,21 @@ class State : FrameLayout {
     }
 
     fun setContentView(view: View) {
-        removeAllViews()
+        removeView(contentView)
         addView(view)
     }
 
-    override fun addView(child: View?) {
-        removeAllViews()
-        super.addView(child)
+    fun setContentView(@LayoutRes layoutRes: Int) {
+        removeView(contentView)
+        val layoutInflater = LayoutInflater.from(context)
+        addView(layoutInflater.inflate(layoutRes, this, false))
```

Comment:
```suggestion
        layoutInflater.inflate(layoutRes, this, true)
```

Reply:
I refractored it, in another commit, so I won't be able to use your suggestion.